plugins/mcreator-localization/lang/texts.properties

```diff
@@ -3097,7 +3097,7 @@ validators.registry_name.invalid={0} can only contain lowercase English letters,
 validators.registry_name.invalid2={0} can''t end with a colon character
 validators.registry_name.invalid3={0} can''t contain more than one colon character
 validators.unique_name.empty={0} can''t be empty
-validators.unique_name.duplicate=This name is already used
+validators.unique_name.duplicate=The specified {0} is already used
```

Comment:
I would use either

`The specified name "{0}" is already used`

or

`Name "{0}" is already used`

or

`"{0}" is already used`

Reply:
Ok it's done